### DISCUSSION TITLE
Can the 'ComparingValue' for CompareConverter be dynamic or bound via binding instead of Static?

### DISCUSSION THREAD

**Discussion by roysurles** [CommunityToolkit/Maui#1224] [General]
Can the 'ComparingValue' for CompareConverter be dynamic or bound via binding instead of Static?

When I try to make the ComparingValue binding to a property, get the following error:
No property, BindableProperty, or event found for "ComparingValue", or mismatching type between value and property.

**Comment by bijington** [comment]
No the `ComparingValue` does not currently support bindings. It sounds like this is something you need? If so are you happy to open a [proposal](https://github.com/CommunityToolkit/Maui/issues/new?assignees=&labels=new%2Cproposal&projects=&template=open-a-new-feature-proposal.yml&title=%5BProposal%5D+) to initiate this change?

**Comment by GeorgeLeithead** [comment]
Can I also suggest that you might want to look at some of the `Converters` that we have availabel in the MCT: [src/CommunityToolkit.Maui/Converters](https://github.com/CommunityToolkit/Maui/tree/81741d913a2cce96ab74c27c541f269bf51c2820/src/CommunityToolkit.Maui/Converters)

**Comment by CliffAgius** [comment]
Closed and answered
